c/include/libsbp/legacy/user.h

```diff
@@ -42,4 +42,4 @@ typedef struct SBP_ATTR_PACKED {
 
 SBP_PACK_END
 
-#endif /* LIBSBP_USER_MESSAGES_H */
\ No newline at end of file
+#endif /* LIBSBP_LEGACY_USER_MESSAGES_H */
```

Comment:
I had a script which would take care of header guards automatically, might be worth resurrecting that especially as we're in the process of massively increasing the number of header files. Using just a linter is pretty annoying for such a trivial issue, a tool which just does it would be preferable to my mind.